.github/.OwlBot.yaml

```diff
@@ -29,3 +30,7 @@ deep-copy-regex:
   dest: "/owl-bot-staging/v2/gapic-google-cloud-storage-v2/src"
 - source: "/google/storage/v2/google-cloud-storage-v2-java/gapic-google-cloud-storage-v2-java/src/main/java/com/google/storage/v2/gapic_metadata.json"
   dest: "/owl-bot-staging/v2/gapic-google-cloud-storage-v2/src/main/resources/com/google/storage/v2/gapic_metadata.json"
+- source: "/google/storage/control/v2/.*-java/proto-google-.*/src"
+  dest: "/owl-bot-staging/v2/proto-google-cloud-storage-control-v2/src"
+- source: "/google/storage/control/v2/.*-java/gapic-google-.*/src"
+  dest: "/owl-bot-staging/v2/google-cloud-storage-control/src"
```

Comment:
@frankyn This "google-cloud-storage-control" is the destination for the GAPIC library, right?

Reply:
Yes, we want to use artifact name as `google-cloud-storage-control`.

Reply:
This is directory name. I think pom.xml will follow the name but let's ensure once the OwlBot pull request comes after merging this.